tests/update.bats

```diff
@@ -69,7 +68,47 @@ load '/usr/local/lib/bats/load.bash'
 
   stub docker \
     "pull my-image : echo pulled image" \
-    "run --interactive --tty --rm --volume /plugin:/bundle_update --volume /plugin/hooks/../update:/update --workdir /bundle_update --env BUNDLE_APP_CONFIG=/bundle_app_config  --env PRE_BUNDLE_UPDATE= my-image /update/update.sh : echo bundle updated"
+    "run --interactive --tty --rm --volume /plugin:/bundle_update --volume /plugin/hooks/../update:/update --workdir /bundle_update --env BUNDLE_APP_CONFIG=/bundle_app_config  --env PRE_BUNDLE_UPDATE= --env POST_BUNDLE_UPDATE= my-image /update/update.sh : echo bundle updated"
+  stub git "diff-index --quiet HEAD -- Gemfile.lock : exit 1"
+  stub buildkite-agent "meta-data set bundle-update-plugin-changes true : echo meta-data set"
+
+  run $PWD/hooks/command
+
+  assert_success
+  assert_output --partial "pulled image"
+  assert_output --partial "bundle updated"
+  unstub docker
+  unstub git
+  unstub buildkite-agent
+}
+
+@test "Passes the pre-bundle-update command" {
```

Comment:
I added in an extra test for the pre-bundle-update command, as well as the post-bundle-update command below.

Reply:
Just so I understand this test clearly.. You're stubbing the docker command.. then running the `hooks/command` and the test is that if the command calls docker with those arguments? (i.e essentially passing in the exported env values)

Reply:
Yeah, exactly. It's not the best test under the sun.

Reply:
I'd take this test over nothing any day 👍